docs/reference/mapping/removal_of_types.asciidoc

```diff
@@ -376,7 +378,7 @@ documents of different types which have conflicting IDs:
 
 [source,js]
 ----
-PUT new_twitter?include_type_name=true
+PUT new_twitter
```

Comment:
should we remove `_doc` type on line 384?

Reply:
same reason as before